src/relay/pass/type_solver.cc

```diff
@@ -404,6 +430,10 @@ bool TypeSolver::Solve() {
       args.push_back(Resolve(tlink->value->FindRoot()->resolved_type));
       CHECK_LE(args.size(), rel->args.size());
     }
+
+    CHECK(rnode->location.defined()) << "undefined location";
```

Comment:
Is every `RelationNode` attached with defined `location`? It looks that CI fails here.

Reply:
Yes they are, fixed the issue.